docs-content/getting-started/fullstack-frameworks/next-js.md

```diff
@@ -77,6 +77,7 @@ export default function MyApp({ Component, pageProps }: AppProps) {
 	return (
 		<>
 			<HighlightInit
+				excludedHostnames={['localhost']}
```

Comment:
in general i'd oppose adding more configuration for a subset of customers, but in this case we've had enough folks to ask for this to warrant building this in.

Reply:
Yeah. I'd expect 100% of our clients to implement some sort of `localhost` block, if only to clean up their own logs.

Reply:
If we did this, won't this mean their sessions won't show up when they install Highlight for the very first time? Is this a good idea?

Reply:
The `excludedHostnames` props defaults to an empty array, so they have to opt-in.

Nobody wants their `localhost` error logs streaming to Highlight in the long-term. It's important to add it to the docs so that users don't have to hunt for it or do it wrong, as we noticed in a recent bug report.

Reply:
I'll add a comment to remind users to remove `localhost` from `excludedHostnames` when testing.

Reply:
sounds good, makes sense.